src/lib/schema.ts

```diff
@@ -33,6 +33,8 @@ export type HydrantState = {
   totalOptions: number;
   units: number;
   hours: number;
+  hoursFirstHalf: number;
+  hoursSecondHalf: number;
```

Comment:
what do you think about the type

```ts
hours: { firstHalf: number; secondHalf: number; }
```

Reply:
and omit the max hours, which i think we can recompute in the thing when we need to? unless we use it somewhere else